docs/standard/base-types/grouping-constructs-in-regular-expressions.md

```diff
@@ -14,461 +14,434 @@ helpviewer_keywords:
+
+You can access named captured groups in the following ways:
+
+- By using the named backreference construct within the regular expression. The matched subexpression is referenced in the same regular expression by using the syntax `\k<`*name*`>`, where *name* is the name of the captured subexpression.
+
+- By using the backreference construct within the regular expression. The matched subexpression is referenced in the same regular expression by using the syntax `\`*number*, where *number* is the ordinal number of the captured subexpression. Named matched subexpressions are numbered consecutively from left to right after matched subexpressions.
+
+- By using the `${`*name*`}` replacement sequence in a <xref:System.Text.RegularExpressions.Regex.Replace%2A?displayProperty=nameWithType> or <xref:System.Text.RegularExpressions.Match.Result%2A?displayProperty=nameWithType> method call, where *name* is the name of the captured subexpression.
+
+- By using the `$`*number* replacement sequence in a <xref:System.Text.RegularExpressions.Regex.Replace%2A?displayProperty=nameWithType> or <xref:System.Text.RegularExpressions.Match.Result%2A?displayProperty=nameWithType> method call, where *number* is the ordinal number of the captured subexpression.
+
+- Programmatically, by using the <xref:System.Text.RegularExpressions.GroupCollection> object returned by the <xref:System.Text.RegularExpressions.Match.Groups%2A?displayProperty=nameWithType> property. The member at position zero in the collection represents the entire regular expression match. Each subsequent member represents a matched subexpression. Named captured groups are stored in the collection after numbered captured groups.
+
+- Programmatically, by providing the subexpression name to the <xref:System.Text.RegularExpressions.GroupCollection> object's indexer (in C#) or to its <xref:System.Text.RegularExpressions.GroupCollection.Item%2A> property (in Visual Basic).
+
+A simple regular expression pattern illustrates how numbered (unnamed) and named groups can be referenced either programmatically or by using regular expression language syntax. The regular expression `((?<One>abc)\d+)?(?<Two>xyz)(.*)` produces the following capturing groups by number and by name. The first capturing group (number 0) always refers to the entire pattern.
+
+| Number | Name             | Pattern                            |
+|--------|------------------|------------------------------------|
+| 0      | 0 (default name) | `((?<One>abc)\d+)?(?<Two>xyz)(.*)` |
+| 1      | 1 (default name) | `((?<One>abc)\d+)`                 |
+| 2      | One              | `(?<One>abc)`                      |
+| 3      | Two              | `(?<Two>xyz)`                      |
+| 4      | 4 (default name) | `(.*)`                             |
```

Comment:
This table is the only place something significant was changed in the file (i.e. the order of the capturing groups).